### DISCUSSION TITLE
Total and sub total decimal precision in tree view

### DISCUSSION THREAD

**Discussion by habibamouri** [OCA/odoo-community.org#87] [Q&A]
I change the decimal precision in setting, technical, decimal accuracy. I change decimal place in account, setting, currency. theses changes have no effect on display subtotal and total decimal precision in tree-view. I would like to have 3 decimal places for total and sub total in tree view for all odoo modules. (version 14, 15 and 16)

**Comment by Shide** [comment]
Total, Subtotal and Taxes decimal places, are related to the currency because it goes on an invoice or orders.
Since these fields are monetary (this means that could be paid in coins of the selected currency), are restricted to the decimal places of the currency.

You can't show a total of 10,001 if your currency (Euros, for example), doesn't have any coin for the value of 0,001. 
This will be a huge mistake and the Tax agency of your country for sure will tell you that you can't show more that number of decimals of your currency.

**Reply by habibamouri** [reply]
Good morning Shide,
Thank you for your interest.
I join 2 fig to explain more my request.
![fig1](https://user-images.githubusercontent.com/17685302/236667527-ce648000-42f8-4717-b1fd-c782025b1606.png)
![fig2](https://user-images.githubusercontent.com/17685302/236667545-0ba49bcc-469d-48d6-a586-515bf801da11.png)
the problem is decimal precision presented in red rectangle

**Comment by Shak88** [comment]
to change the decimal accuracy to 3 digits go to Accounting > Configuration > Accounting > Currencies.
then change the rounding factor from 0.01 to 0.001

hope this helps

**Reply by habibamouri** [reply]
As you can see in the rows of the tree view values have 3 digits. This result after changing the rounding factor from 0.01 to 0.001 and the accuracy. My request is for the end line marked with red squares (in fig1 and fig2) and the first line (subtotal of group in fig 1).

**Comment by Shide** [comment]
Ah, I see... Try to display the field `amount_total_signed` (this fields has the company currency and not the invoice). Maybe the numbers look closest to the real amount.

If you want to see the real amount, you should go to the `account.move` menu and check in it.
